src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferAgent.java

```diff
@@ -99,6 +102,10 @@ public StageInfo getStageInfo() {
     return this.stageInfo;
   }
 
+  public String getQueryID() {
```

Comment:
I feel this method is not used. Am I right?

Reply:
good point! Removed